source/user-manual/capabilities/vulnerability-detection/allow_os.rst

```diff
@@ -0,0 +1,125 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _vu_allow_os:
+
+Scan vulnerabilities on unsupported systems
+===========================================
+
+There are many operating systems derived from others that Vulnerability Detector supports :doc:`Vulnerability Detector supports<compatibility_matrix>` to
+which compatibility of vulnerability feeds could be extended, such as Linux Mint and Oracle Linux. However, it is not possible to automate the degree
+of acceptance of feeds for these systems, so their vulnerabilities are not scanned by default.
```

Comment:
This sentence is a little bit confusing. Look for another way to say it, such as, `However, those distributions are not scanned by default since there are not pre-defined feeds for them.`